vllm/distributed/kv_events.py

```diff
@@ -36,8 +36,10 @@ class KVCacheEvent(
         array_like=True,  # type: ignore[call-arg]
         omit_defaults=True,  # type: ignore[call-arg]
         gc=False,  # type: ignore[call-arg]
-        tag=True):
+        tag=True,
+        kw_only=True):
     """Base class for all KV cache-related events"""
+    metadata: Optional[dict[str, Any]] = None
```

Comment:
@vMaroon adding metadata here kind of defeats the purpose of having the strongly-typed event types. Could you elaborate on what it's for?

Reply:
I don't see any use for this right now - static metadata information should be discovered and not mounted per event. @orozery, if there is no concrete sensible immediate use for this field, can you eject?